.browserslistrc

```diff
@@ -0,0 +1,13 @@
+# https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices
```

Comment:
Nice little touch including this. Do we want to add a quick one-liner about how/why our compat list may differ from the service manual?

I'm thinking of the (exceedingly unlikely?) event that a future dev has a look at this without knowing the history and updates it to match the service manual completely.

Reply:
@domoscargin Ahh you'll have to teach me as well 😊 

Can you offer a suggestion? If not I can leave this line out for another PR (can add later)

Thanks

Reply:
Something like "This list builds on the GOV.UK service manual's browser testing recommendations (https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices)"?